examples/05_joining_tables_with_FuzzyJoin.py

```diff
@@ -110,6 +115,11 @@
 # To join them with dirty_cat, we only need to do the following:
 from dirty_cat import fuzzy_join
 
+# We will ignore the warnings:
+import warnings
+
+warnings.filterwarnings("ignore")
```

Comment:
If it's just the warning on experimental don't catch it.

Reply:
Yes that's the reason!